src/model.js

```diff
@@ -84,6 +93,9 @@ class Model {
         self._db[util.COLLECTION_TYPE.VOICE] = new Datastore(db.voice);
         self._db[util.COLLECTION_TYPE.CREDS] = new Datastore(db.credentials);
         self._db[util.COLLECTION_TYPE.EMAIL] = new Datastore(db.emails);
+        self._db[util.COLLECTION_TYPE.COMMAND_REPORT] = new Datastore(db.commandReport);
```

Comment:
Just to clarify, did you decide to keep the metrics in separate collections or are you still working on condensing them into one?